priv/package.json

```diff
@@ -36,7 +36,14 @@
     "mocha": "3.2.0",
     "style-loader": "0.13.0",
     "url-loader": "0.5.7",
-    "webpack": "1.12.9"
+    "webpack": "1.14.0",
+    "react-bootstrap": "0.30.7",
```

Comment:
Starting from here - it should be put in `dependencies` not in `devDependencies`.
Also it could be sorted alphabetically.